ansys/content.jsonc

```
@@ -2,22 +2,22 @@
     "name": "Ansys",
     "category": "Simulation",
     "logo_path": "../_icons/Ansys.svg",
-    "short_desc": "Model how infrastructure is affected by various hazards.",
+    "short_desc": "Model how various hazards affect infrastructure",
```

Adding period like others I see on production.

```suggestion
    "short_desc": "Model how various hazards affect infrastructure.",
```